README.md

```diff
@@ -22,10 +22,17 @@ from gi.repository import Modulemd
 The libmodulemd API provides a number of convenience tools for interacting
 with repodata (that is, streams of YAML that contains information on multiple
 streams, default data and translations). The documentation will use two
-repositories, called "fedora" and "updates" for demonstrative purposes. It
-will assume that the content of the YAML module metadata from those two
-repositories have been loaded into string variables "fedora_yaml" and
-"updates_yaml", respectively.
+repositories, called "fedora" and "updates" for demonstrative purposes. 
```

Comment:
You can eliminate this section entirely. The original text should have been fine. I intentionally didn't show the real data here because the YAML for repodata is quite long (thousands of lines).